src/components/SaveTimeRecording.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import { saveTime } from "../Modules/SaveTimesheet";
+import TimeInputForm from "./TimeInputForm";
+import { Form, Button, Grid } from "semantic-ui-react";
+
+class SaveTimeRecording extends Component {
+	state = {
+		begin: "",
+		end: "",
+		saveTime: false
```

Comment:
I would change this to something more descriptive like `timeSaved`. That would make more sense due to how you use that state.